.github/workflows/canary.yml

```diff
@@ -11,6 +11,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@master
+
+      - name: Embed version
+        run: |
+          echo " cachedversion = \"Canary ${GITHUB_SHA:0:7}\"" >> lua/wire/server/wirelib.lua
```

Comment:
I'd prefer a newline over a single space which seems like a mistake rather than necessary to avoid a syntax error